docs/wiki/EL/el-clients.md

```diff
@@ -1,60 +1,98 @@
-# Execution Layer Implementations
+# Execution Client
 
-> :warning: This article is a [stub](https://en.wikipedia.org/wiki/Wikipedia:Stub), help the wiki by [contributing](/contributing.md) and expanding it.
+> **Execution clients**, formerly known as *eth1 clients*, are tasked with processing and broadcasting transactions and managing Ethereum's state.
+They run the computations for each transaction using the [Ethereum Virtual Machine](https://ethereum.org/en/developers/docs/evm/) to ensure that the rules of the protocol are followed.
+Execution clients can be configured as archive nodes, which retain the entire history of the blockchain, or as full (pruned) nodes, which keep only the latest state and discard older data to save storage space.
 
-Resources covering all current and historical execution clients. Overview of client unique features of each client, architecture, guides and resources.
 
-## Clients in production
+## Overview Table
+
+| Client                                                          | Language   | Developer           | Status      |
+| --------------------------------------------------------------- | ---------- | ------------------- | ----------- |
+| [Besu](https://github.com/hyperledger/besu)                     | Java       | Hyperledger         | Production  |
+| [Erigon](https://github.com/ledgerwatch/erigon)                 | Go         | Ledgerwatch         | Production  |
+| [Geth](https://github.com/ethereum/go-ethereum)                 | Go         | Ethereum Foundation | Production  |
+| [Nethermind](https://github.com/NethermindEth/nethermind)       | C#         | Nethermind          | Production  |
+| [Reth](https://github.com/paradigmxyz/reth)                     | Rust       | Paradigm            | Production  |
+| [Nimbus](https://github.com/status-im/nimbus-eth1)              | Nim        | Nimbus              | Development |
+| [Silkworm](https://github.com/erigontech/silkworm)              | C++        | Erigon              | Development |
+| [JS Client](https://github.com/ethereumjs/ethereumjs-monorepo)  | Typescript | Paradigm            | Development |
+| [Akula](https://github.com/akula-bft/akula)                     | Rust       | Akula Developers    | Deprecated  |
```

Comment:
Maybe I would split the table here to have clear overview of current and historical clients. At first glance it just feels weird to have it all in one place but maybe it works just with Status column, I'll let others comment on this.

Reply:
Btw there is also a rust execution client in development by lambdaclass https://github.com/lambdaclass/ethereum_rust